python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -575,6 +575,7 @@ def convert_pooling(node, **kwargs):
     pool_type = attrs["pool_type"]
     stride = eval(attrs["stride"]) if attrs.get("stride") else None
     global_pool = get_boolean_attribute_value(attrs, "global_pool")
+    p_value = int(attrs.get('p_value', '2'))
```

Comment:
why p_value is default to 2? MXNet's default seems to be None. and it can be 1 or 2

Reply:
shape can be [] which would mean that one sample is drawn

Reply:
fixed this